test/parallel/test-whatwg-webstreams-adapters-streambase.js

```diff
@@ -65,3 +65,12 @@ const {
   const readable = newReadableStreamFromStreamBase(stream);
   readable.cancel().then(common.mustCall());
 }
+
+{
+  const stream = new JSStream();
+  stream.onread = common.mustCall(() => {});
```

Comment:
To fix the linter error
```suggestion
  stream.onread = common.mustCall();
```

Reply:
Done.